src/modules/colorPicker/ColorPickerUI/Keyboard/KeyboardMonitor.cs

```diff
@@ -87,20 +86,12 @@ private void Hook_KeyboardPressed(object sender, GlobalKeyboardHookEventArgs e)
                     _currentlyPressedKeys.Add(name);
```

Comment:
if you press left Ctrl, this will add "Ctrl (left)" because you removed that "clean up" code above, it will also add "Ctrl" from AddModifierKeys, so now you have 2 keys pressed - "Ctrl (left)" and "Ctrl"

Reply:
This was done on purpose to fix the issue where the non-modifier key is not pressed last.

Reply:
Then you should not add it here at all, it just creates a confusing logic when there will be 2 buttons pressed when there is only one pressed.

Reply:
That would imply writing more code which does nothing behavior-wise. Instead, I added a comment explaining the logic.